### ISSUE TITLE
BFB comparison tests for pace

### ISSUE THREAD

**Issue by fmalatino** [NOAA-GFDL/pace#91]
**Is your feature request related to a problem? Please describe.**
A bit for bit (BFB) comparison test needed to ensure incoming changes do not effect reproduciblity  of output during runs.

**Describe the solution you'd like**
An regression test using [`cprnc`](https://github.com/ESMCI/cprnc) for BFB comparison of output from a run before and after changes to be merged.

**Comment by xyuan** [comment]
yeah, I vote for it, this bfb labels help us to trace pr issues in the future
